src/Language/Fixpoint/Congruence/Types.hs

```diff
@@ -39,28 +39,28 @@ data Disequality = Diseq Term Term
 --------------------------------------------------------------------------------
 -- | Exported Constructors
 --------------------------------------------------------------------------------
-app :: F.Symbol -> [Term] -> Term
+app :: F.Symbol s -> [Term] -> Term
 app f as = intern (BApp f as)
 
-var :: F.Symbol -> Term
+var :: F.Symbol s -> Term
 var x = intern (BVar x)
 
 --------------------------------------------------------------------------------
 -- | Hash-consed Term DataType
 --------------------------------------------------------------------------------
 data Term 
```

Comment:
How do these compile without adding an `s` to `Term`?

Reply:
src/Language/Fixpoint/Congruence/Types.hs is not included in the .cabal file. It doesn't typecheck but stack won't throw any error since it never gets compiled..

Reply:
Ok. Let's leave it for now and we can ask if it's needed.